src/components/quiz/QuizConstructor.vue

```diff
@@ -117,28 +98,53 @@ function deleteQuestion(index: number) {
   quizList.value.splice(index, 1);
 }
 
-function setCandidateId(id: number) {
+function setCandidate(id: number, name: string) {
   currentCandidateId.value = id;
+  currentCandidateName.value = name;
+}
+
+async function postPercentageResult() {
+  const percentageResult = ref<PercentsResult>({
+    candidateUsername: "",
+    resultPoints: 0,
+  });
+  percentageResult.value.candidateUsername = currentCandidateName.value;
+  percentageResult.value.resultPoints = resultPercents.value;
+
+  try {
+    await resultsStore.postPercentageResult(percentageResult.value);
+  } catch (e) {
+    console.log(e);
+  }
 }
 
-function postResult() {
+async function postResult() {
   const result = ref<QuizResult>({
     questionAnswer: [],
-    id: 0,
+    title: "",
     startedAt: 0,
     endedAt: 0,
   });
   result.value.questionAnswer = quizList.value;
-  result.value.id = currentCandidateId.value;
+  result.value.title = `candidate id ${
```

Comment:
can we keep this in one line? Variables in string interpolations usually are not wrapped as objects